eclair-core/src/test/scala/fr/acinq/eclair/EclairImplSpec.scala

```diff
@@ -429,4 +430,66 @@ class EclairImplSpec extends TestKitBaseClass with FixtureAnyFunSuiteLike with I
     assert(expectedPaymentPreimage === ByteVector32(keySendTlv.value))
   }
 
+  test("sign an arbitrary message with the node's private key") { f =>
```

Comment:
I think this test and the next one are redundant.
They don't even need to inject a custom `KeyManager` with a hard-coded seed to test what you want.
You can group them in a single test that does the following:

- Use the default `KeyManager` (no need to inject a custom seed)
- Sign a message with `signMessage`
- Verify it with `verifyMessage`
- Ensure that the recovered `signerNodeId` matches `kit.nodeParams.nodeId` and verify that the signature is correct (with "normal" signature validation with `Crypto.verifySignature`)

Reply:
Done